syntax = "proto3";
package tutorial;

 
enum TaskResult 
  {
    FAIL = 0;
    SUCCESS = 1;
  }

message Agent {
  int32 task_id = 1;
  string task_uuid = 2;
  TaskResult result = 3;
  string message= 4;
}
